### PR TITLE
Website cosmetic changes

### LINKED ISSUE #118: Banner Title Change
Banner title style changed to gradient -   
Add drop shadow : text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
font : H1 ;
font-weight: 300;
line-height: normal;
letter-spacing: -1.44px;
background: linear-gradient(90deg, #3F6BAE 3%, #6DA39F 33.33%, #D89E37 66.67%, #823628 100%);
@alexandercyu

### LINKED ISSUE #120: Fix alignment - Quick start menu
Fix alignment for the Quick start menu and adjust opacity 

opacity: 0.4;
background: url(<path-to-image>) lightgray -126.294px 0px / 147.687% 121.823% no-repeat, rgba(0, 0, 0, 0.50);

/* Elevation 3 */
box-shadow: 0px 4px 8px 3px rgba(0, 0, 0, 0.15), 0px 1px 3px 0px rgba(0, 0, 0, 0.30);

Use layer with 50% opacity 
Quickstart-layer

@alexandercyu

### LINKED ISSUE #124: Change the Marble Title on mobile
To align with the title on browser site, change banner title for mobile site. 
The Title font will be changed from H1 to H2 , colour scheme remains the same. Drop shadow not required as there is no video behind the text

### PR DESCRIPTION
**Pull request by alexandercyu**
Fixes #118 
Fixes #120 
Fixes #124
- Add drop shadow to banner text
- Changed padding around quickstart menu items to 30px
- Changed space  between quickstart underline and first menu item to 25px
- Added black opaque background to quickstart menu

**2024-05-10**
- Added h2 version of main heading for smaller screens
- Added changes from reviews

### CODE REVIEW

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

**Review by ShrutiK100 — COMMENTED** [review]
Spacing below forum needs to be removed.
Change padding to 40px
Current dimensions for Quickstart menu- 15px below "Quickstart" title , above the line.
25 px between the Title+line and menu items
20 px between each menu item 
Menu box should be Width-500px and height-400px

File: static/css/marble.css [hunk 1]
```diff
@@ -16,14 +16,20 @@
...
+
     /*Colours*/
     --alert-background: #F6DC3D;
     --background: #FFFFFF;
     --background-black: #000000;
+    --transparent: transparent;
     --background-transparent: transparent;
     --black: #000000;
-    --opaque-black: rgba(0,0,0,0.5);
+    --black-opacity-0-30: rgba(0,0,0,0.30);
```
[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/marble.css:29]: Renamed these to be more general.

File: static/css/marble.css [hunk 2]
```diff
@@ -16,14 +16,20 @@
...
     --elevation-4: 0px 6px 10px 4px rgba(0, 0, 0, 0.15), 0px 2px 3px 0px rgba(0, 0, 0, 0.30);
 
+    /*Shadows*/
+    --text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+
     /*Colours*/
     --alert-background: #F6DC3D;
     --background: #FFFFFF;
     --background-black: #000000;
+    --transparent: transparent;
```
[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/marble.css:26]: It just seemed weird using a "background-" variable for non background use.  But I guess that doesn't matter under the hood.

Removed this variable.

**Comment by alexandercyu** [comment]
> Spacing below forum needs to be removed. Change padding to 40px Current dimensions for Quickstart menu- 15px below "Quickstart" title , above the line. 25 px between the Title+line and menu items 20 px between each menu item Menu box should be Width-500px and height-400px

Changes made.

**Comment by alexandercyu** [comment]
> > Fixes #118 #120
> > 
> > * Add drop shadow to banner text
> > * Changed padding around quickstart menu items to 30px
> > * Changed space  between quickstart underline and first menu item to 25px
> > * Added black opaque background to quickstart menu
> 
> Each menu item has its own adjustments. Please apply these adjustments on each menu item. display: flex; width: 192px; height: 47px; flex-direction: column; justify-content: center; flex-shrink: 0;
> 
> When you will click on each menu item in developer mode you will see code that needs to be applied on Menu title and each menu item. Same has be shared in the screenshot. Screenshot 2024-05-09 at 9 49 09 AM

Applied these changes.

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
Have commented on the changes required

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Add some default space between the title and the quickstart menu. Right now, they are right up against each other when the screen width is small:

image

**Comment by alexandercyu** [comment]
> Have commented on the changes required

I can't see your comments.  Did you make them in the files?

**Comment by alexandercyu** [comment]
> Add some default space between the title and the quickstart menu. Right now, they are right up against each other when the screen width is small:
> 
> image

Added 50px space between them

**Review by ShrutiK100 — COMMENTED** [review]
Screenshot 2024-05-14 at 10 37 58 AM
Not sure why in the mobile view (screenshot shared) the Title "MARBLE" is placed on the left of the quick start menu. The tile needs to be above the Quickstart menu according to mobile design.

**Comment by mishaschwartz** [comment]
@alexandercyu I'll review after conflicts are resolved

**Comment by alexandercyu** [comment]
> @alexandercyu I'll review after conflicts are resolved

The Resolve conflict button is greyed out for me.  Do I have to resolve this locally?

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
You're missing a closing brace somewhere in marble.css so the css file isn't loading properly.

**Comment by alexandercyu** [comment]
> You're missing a closing brace somewhere in marble.css so the css file isn't loading properly.

Fixed the missing brace.

**Review by mishaschwartz — APPROVED** [review]

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
Add 50px between the Marble title and Quickstart menu . Remove the drop shadow on Marble text(only for mobile design
Screenshot 2024-05-17 at 4 02 10 PM
)

**Comment by alexandercyu** [comment]
> Add 50px between the Marble title and Quickstart menu . Remove the drop shadow on Marble text(only for mobile design Screenshot 2024-05-17 at 4 02 10 PM )
> 


Are you saying remove the drop shadow from the banner title and all quickstart menu text for mobile size only?  Keep the drop shadow on desktop sizes?

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
Looks good in mobile mode although in tablet screen size browser the title "Marble" is set to white colour. Please change it to the standard colour applied in mobile and browser.
Screenshot 2024-05-21 at 9 15 20 AM

Also noticed a text drop occurring on tablet screen size. 
Screenshot 2024-05-21 at 9 24 28 AM

**Comment by alexandercyu** [comment]
> Looks good in mobile mode although in tablet screen size browser the title "Marble" is set to white colour. Please change it to the standard colour applied in mobile and browser. Screenshot 2024-05-21 at 9 15 20 AM
> 
> Also noticed a text drop occurring on tablet screen size. Screenshot 2024-05-21 at 9 24 28 AM

- Made the Marble title rainbow coloured like in the design.
- Fixed the quickstart title wrapping when transitioning between screen sizes
- Made Marble title h1 in tablet size.

**Review by ShrutiK100 — APPROVED** [review]
Looks good!

**Pull request merged**